httpserver/http_connection.cpp

```diff
@@ -357,6 +399,34 @@ void connection_t::process_request() {
             response_.result(http::status::ok);
             service_node_.process_push_batch(request_.body());
 
+        } else if (target == "/msg_test") {
```

Comment:
We should also verify the signature and pubkey from the snode who made the request.
(guard with the same `#ifdef` as the other snode to snode requests)

Reply:
Updated